.bazelrc

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Import configuration for Remote Builds
```

Comment:
Can we just include the RBE config directly in this file, to keep the top level directory cleaner and have only one place for configuration? Or move rbe.bazelrc somewhere under build_tools. You can add a block comment separating the sections if you want (e.g. https://github.com/tensorflow/tensorflow/blob/master/.bazelrc#L307-L309).

Reply:
We could just include them directly. I thought it was nice to stick them in a separate file. Happy to put it somewhere else though.

`build_tools/bazel/rbe.bazelrc` ?
`build_tools/bazel/bazelrc/rbe.bazelrc` ?
`build_tools/rbe.bazelrc`

We've already got `build_tools/bazel_build.sh`. If I made `build_tools/bazel` I could move that there as well.

Reply:
Hmmm.. I vote for directly including them. I don't have a particularly strong opinion though, except that I'd prefer to keep the top level directory as simple as possible (on that note, `kokoro/` could move to `ci/` :D

Reply:
I like modularity, personally. I dunno, I'm someone who has like 15 .bashrc files that all get sourced into one file though. Stella voted to include them directly as well though. Updated

Re: Moving kokoro/ to ci/ would still be the same number of directories. If we create other CI stuff we can move kokoro in there. For now though, the stuff in there really is kokoro-specific and we have other CI stuff in .github/workflows, so I don't think that makes a ton of sense